app/controllers/concerns/jsonapi.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Concerns
+  module Jsonapi
+    extend ActiveSupport::Concern
+
+    include JSONAPI::Pagination
+    include JSONAPI::Deserialization
+
+    private
+
+    def jsonapi_serializer_params
+      {
+        current_user: current_user
+      }
+    end
+
+    # def jsonapi_serializer_class(resource, is_collection)
```

Comment:
what would this be for? and the obvious question... why do we keep it commented?

Reply:
It's not being used currently, but it's to define our custom way to determine the serializer class for a given object.